public/services/IndexService.ts

```diff
@@ -26,25 +27,25 @@ export default class IndexService {
   }
 
   getIndices = async (queryParamsString: string): Promise<ServerResponse<GetIndicesResponse>> => {
-    const response: IHttpResponse<ServerResponse<GetIndicesResponse>> = await this.httpClient.get(
-      `../api/ism/_indices?${queryParamsString}`
-    );
+    const url = `..${NODE_API._INDICES}?${queryParamsString}`;
+    const response = (await this.httpClient.get(url)) as IHttpResponse<ServerResponse<GetIndicesResponse>>;
```

Comment:
nit: no need to create local variable for response unless we will use it, e.g. logging
we could return {}.data directly.

Reply:
I generally agree, I just find this hard to read:
```
return ((await this.httpClient.get(url)) as IHttpResponse<ServerResponse<GetIndicesResponse>>).data;
```